### PR TITLE
[ENGINEERING-547] Update readme & add license file

### PR DESCRIPTION
**Pull request by tkanzakic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+    const environment = 'production'
+    aptoInitialiseSDK(apiKey, userToken, cardId, lastFour, environment, 'apto-pci-sdk', callbackFunction)
+  }
+</script>
+<body onload="onLoad()">
+  <div id="apto-pci-sdk"></div>
+</body>
+```
+
+this call accept a callback function that will be invoked once the SDK has been initialised.
```
Comment (ioliverm) [inline README.md]: `accept` -> `accepts`

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+          'position: absolute; bottom: 22px; left: 22px; color: white; font-family: monospace; font-size: 16px; text-shadow: 0 1px 1px rgba(43, 45, 53, 0.3);'
+      }
+    }
+    aptoCustomiseCardAppearance(styling)
+    aptoShowCardLastFour()
+  }
+</script>
+```
+
+The `aptoCustomiseCardAppearance` method accept accept a valid CSS for the four customisable elements: card background (container), pan, cvv and expiration date (exp).
```
Comment (ioliverm) [inline README.md]: `accept` -> `accepts`

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+
+## Initialising the SDK
+
+We recommend initialising the SDK once the page where it will be shown has finished loading:
+
+```html
+<script>
+  function onLoad() {
+    const apiKey = ''
+    const userToken = ''
```
Comment (ioliverm) [inline README.md]: We should add a chapter to explain that:

1: apiKey is the Mobile API public key
2: cardId and lastFour have to be obtained previously (Mobile API or Core API)
3: userToken must be obtained using the verifications + login endpoints in the Mobile API (maybe a link to the mobile api specs here?)

File: README.md (hunk 1 above)
Comment (tkanzakic) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (tkanzakic) [reply README.md]: fixed

**Pull request merged**